### PR TITLE
[17.0] [MIG] account_payment_partner

### PR DESCRIPTION
**Pull request by miquelalzanillas**
Depends on:

- [#1182]

### CODE REVIEW

**Comment by miquelalzanillas** [comment]
Hi @pedrobaeza , tests are failing for a missing module dependency for which I also have an open PR [#1182](https://github.com/OCA/bank-payment/pull/1182). Do I have to wait now until the PR is merged for push again here to re-run tests?

**Comment by pedrobaeza** [comment]
https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

**Comment by miquelalzanillas** [comment]
Thanks!

**Comment by miquelalzanillas** [comment]
Ready. Please review: @peluko00 @lbarry-apsl @javierobcn

**Review by samoros1 — APPROVED** [review]

**Review by lbarry-apsl — APPROVED** [review]
Validated functionality

File: account_payment_partner/views/account_move_line.xml
Comment (pedrobaeza) [inline account_payment_partner/views/account_move_line.xml]: And the same for similar cases with booleans:

```suggestion
                        readonly="account_type not in ['asset_receivable', 'liability_payable'] or reconciled"
```

Comment (miquelalzanillas) [reply account_payment_partner/views/account_move_line.xml]: Fixed. Thanks.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

File: account_payment_partner/models/account_move.py
Comment (rousseldenis) [inline account_payment_partner/models/account_move.py:115]: @miquelalzanillas @grindtildeath I'm wondering why removing the bank if it is already set and no payment mode ?

**Pull request merged**
